src/test/java/marquez/common/models/CommonModelGenerator.java

```diff
@@ -0,0 +1,99 @@
+package marquez.common.models;
+
+import java.util.Random;
+
+public final class CommonModelGenerator {
+  private CommonModelGenerator() {}
+
+  private static final int BOUND = DatasourceType.values().length;
+  private static final Random RANDOM = new Random();
+
+  public static OwnerName newOwnerName() {
+    return newOwnerNameWith("test_owner" + newId());
+  }
+
+  public static OwnerName newOwnerNameWith(String value) {
```

Comment:
Should we add validation, such as `@NotBlank` or `@NonNull`? Should we also make it final?

Reply:
For test code we can relax input validation. Let's say an invalid `value` is passed to `newOwnerNameWith`, an exception will be thrown by `wnerName.fromString()`. So, we're covered :)